app/views/admin/dashboard/index.html.slim

```diff
@@ -19,7 +19,7 @@
           .col-xs-4
             i.fa.fa-user.users.fa-2x
           .col-xs-8
-            span.fa-3x= User.count
+            span.fa-3x= User.where(enabled: true).count
```

Comment:
I've added a `enabled` scope in the User model, so now it's:
```ruby
User.enabled.count
```
I think it's fine to leave it like this :)